src/scripts/templates/standalone/Traverse sites tree.js

```diff
@@ -3,12 +3,16 @@
 // Standalone scripts have no template.
 // They are only evaluated when you run them. 
 
+// The following handles differences in printing between Java 7's Rhino JS engine
+// and Java 8's Nashorn JS engine
+if (typeof println == 'undefined') this.println = print;
+
 function listChildren(node, level) {
     var i;
-    for (i=0;i<level;i++) print ("    ");
+    for (i=0;i<level;i++)
     var j;
     for (j=0;j<node.getChildCount();j++) {
-        println(node.getChildAt(j).getNodeName());
+        println(Array(i).join("    ") + node.getChildAt(j).getNodeName());
```

Comment:
It could use `level + 1` instead of `i` (and previous for loop could be removed).

This change also fixes an issue in the original script (which would print root nodes in wrong position, i.e. as if it was a sub-node of the previously printed node).